engine/interpreter-dsl-test/src/test/java/org/enso/interpreter/dsl/test/InliningBuiltinsOutNode.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.interpreter.dsl.test;
+
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.junit.Assert;
+
+@BuiltinMethod(type = "InliningBuiltins", name = "out")
+final class InliningBuiltinsOutNode extends Node {
+
+  long execute(VirtualFrame frame, long a, long b) {
+    Assert.assertNotNull("VirtualFrame is always provided " + frame);
```

Comment:
`"X: " + frame` will evaluate to `"X: null"` if `frame == null`, it will never be `null` itself.

I think this test should be:
```suggestion
    Assert.assertNotNull("VirtualFrame is always provided:", frame);
```

Reply:
Fixed in https://github.com/enso-org/enso/pull/6335/commits/102869d266be0690c8daa3bd64c26f636e504d01